README.md

```diff
@@ -109,6 +109,13 @@ scrape_configs:
         replacement: 127.0.0.1:9115  # Blackbox exporter.
 ```
 
+## Permissions
+
+The ICMP probe requires elevated privileges to function:
+
+* *Linux*: root user _or_ `CAP_NET_RAW` capability is required.
+  * Can be set by executing `setcap cap_net_raw+ep blackbox_exporter`
+* *BSD / OS X*: root user is required.
```

Comment:
I had the Windows line still in earlier, but seeing as there isn't a win32 release for blackbox_exporter I wasn't sure if we wanted to imply that the exporter has been tested on Windows or not. Do you want me to include it still?